doc/userdoc/contribute/templates_styleguides/styleguide.rst

```diff
@@ -0,0 +1,608 @@
+The NEST documentation style guide
+==================================
+
+.. contents:: On this page, you'll find
+   :local:
```

Comment:
This list of contents is very long and unstructured. Can we organise it differently?

Reply:
I moved content around and changed the headings to try to better separate the sections. It should be checked if it makes sense. I also limited the contents on the right side bar to a depth of 1 level